nucleus/src/main/java/za/co/absa/subatomic/adapter/team/rest/TeamController.java

```diff
@@ -88,6 +84,23 @@ public TeamController(TeamService teamService) {
                     request.getDevOpsEnvironment().getRequestedBy());
         }
 
+        if (request.getMembershipRequests() != null) {
+            for (MembershipRequestResource membershipRequest : request
+                    .getMembershipRequests()) {
+                if (membershipRequest.getMembershipRequestId() != null &&
```

Comment:
Better to use `StringUtils.isNotBlank(string)` from `org.apache.commons.lang3` to check for blank Strings.
If you have multiple Strings and you want to check if none are blank, then use `StringUtils.isNoneBlank(string, string, ...)`

Reply:
Is there something similar to this for values that are not String's but Objects?

Reply:
That there is: `ObjectUtils.allNotNull(obj, obj, ...)`